projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -322,13 +322,36 @@ private function prepare_menu_item_icon( $icon ) {
 			if ( 0 === strpos( $icon, 'data:image/svg+xml' ) ) {
 				$img = $icon;
 			} elseif ( 0 === strpos( $icon, 'dashicons-' ) ) {
-				$img = sanitize_html_class( $icon );
+				$icon = $this->prepare_dashicon( $icon );
+				$img  = sanitize_html_class( $icon );
```

Comment:
I think this can be moved inside the prepare_dashicon method, and probably unnecessary since we filter the icons based on the hardcoded array.

Reply:
Removed.